src/service/service-directory-api/tests/FamilyHubs.ServiceDirectory.Api.FunctionalTests/WhenUsingServiceApiUnitTests.cs

```diff
@@ -103,19 +105,19 @@ public async Task ThenTheSimpleListOfServicesAreRetrieved()
         var request = new HttpRequestMessage
         {
             Method = HttpMethod.Get,
-            RequestUri = new Uri(Client.BaseAddress + $"api/services-simple{url}")
+            RequestUri = new Uri(Client!.BaseAddress + $"api/services-simple{url}")
         };
 
-        using var response = await Client.SendAsync(request);
+        using var response = await Client!.SendAsync(request);
 
         response.EnsureSuccessStatusCode();
-
+        response.StatusCode.Should().Be(HttpStatusCode.OK);
 
         var retVal = await JsonSerializer.DeserializeAsync<PaginatedList<ServiceDto>>(await response.Content.ReadAsStreamAsync(), new JsonSerializerOptions { PropertyNameCaseInsensitive = true });
-        var item = retVal?.Items.Find(x => x.Name == "Test Service - Free - 10 to 15 yrs");
-
-        response.StatusCode.Should().Be(HttpStatusCode.OK);
         retVal.Should().NotBeNull();
+        retVal?.Items.Count.Should().Be(ActiveServiceCount);
```

Comment:
Do we need null check on retVal ? if the above line has asserted it to be not null. If it is still showing as nullable then ! should be used. Same elsewhere below

Reply:
It's there because the IDE was complaining still. Will swap to `!`